app/dialog/export/codec/h264section.cpp

```diff
@@ -42,6 +42,32 @@ H264Section::H264Section(int default_crf, QWidget *parent) :
   layout->setMargin(0);
 
   int row = 0;
+  layout->addWidget(new QLabel(tr("Encode Speed:")), row, 0);
+
+
+  char presettooltip[] = "<head/><body>This setting allows you to tweak the ratio of export speed to compression quality. <br /> <br />"\
```

Comment:
This is the wrong way to do this. You should be using `tr("string")` instead of declaring a char array. There's also no need for HTML tags. Just use `\n` for newlines instead.

Reply:
Before I mess something else up, code should look like this then? It compiles and looks fine, or are spaces preferred between then newlines?

```
  preset_combobox_->setToolTip(tr("This setting allows you to tweak the ratio of export speed to compression quality. \n\n"\
    " If using Constant Rate Factor, slower speeds will result in smaller file sizes for the same quaity. \n\n"\
    " If using Target Bit Rate or Target File Size, slower speeds will result in higher quality for the same bitrate/filesize. \n\n"\
    " This setting is equivalent to the `preset` setting in libx264."));
```

Reply:
You don't need the spaces in each line, you can just have `"If using Constant Rate Factor` and end with `for the same quality.\n\n"` (also a typo in "quality"). Also you shouldn't need a `\` at the end of the line, the compiler should just combine adjacent string literals.

Reply:
thanks, I've updated the PR. it's hard to juggle languages and different coding styles, so it helps alot

Reply:
Indeed, especially when it comes to strings in C++.

Anyways, this looks good to me. I'll go ahead and merge once the CI is done.